packages/gatsby-source-appwrite/README.md

```diff
@@ -0,0 +1,85 @@
+# gatsby-source-appwrite
+
+<table align="center">
+  <tr>
+    <td valign="top"><img height="100px" width="100px" src="https://res.cloudinary.com/practicaldev/image/fetch/s--Fda3jJaA--/c_fill,f_auto,fl_progressive,h_320,q_auto,w_320/https://dev-to-uploads.s3.amazonaws.com/uploads/organization/profile_image/2653/80840bff-1061-4544-841e-86da7aa1dd8e.png"></td>
+    <td valign="top"> <img height="100px" width="100px" src="https://res.cloudinary.com/practicaldev/image/fetch/s--hsQ1oxcF--/c_fill,f_auto,fl_progressive,h_320,q_auto,w_320/https://dev-to-uploads.s3.amazonaws.com/uploads/organization/profile_image/2225/18b2c66e-fba2-4f4f-8196-8184c599b75d.png"></td>
+  </tr>
+</table>
+
+Gatsby source plugin for building websites using Appwrite as a data source
+
+## How to install
+
+```
+npm install gatsby-source-appwrite appwrite
```

Comment:
Here you say to install `appwrite` but the peer dependency listed in the `package.json` is `node-appwrite`. Should these match?

Reply:
You are correct, I was hoping to use the isomorphic package, but cannot yet and had to fall back to node.